### PR TITLE
ansible 7.1.0 ansible 6.7.0 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by dawidd6** [comment]
Should we also provide `ansible@6` formula right now? Ansible 6 is still supported.

**Comment by chenrui333** [comment]
Works for me, adding a `ansible@6` as well

**Comment by chenrui333** [comment]
added a placeholder `disable` line to reminder to disable after `6.7.0` release

File: Formula/ansible@6.rb [hunk 1]
```diff
@@ -0,0 +1,613 @@
...
+
+  desc "Automate deployment, configuration, and upgrading"
+  homepage "https://www.ansible.com/"
+  url "https://files.pythonhosted.org/packages/06/75/6e1da76cbc323e5170ac9e63b7c86e17f4387614129ef2514b407c6080d2/ansible-6.6.0.tar.gz"
+  sha256 "e1b940a8d4f412123ede3c14b25cb99c3c8a4d535fd040aabf8e4fb7b0e4f092"
+  license "GPL-3.0-or-later"
+
+  keg_only :versioned_formula
+
+  # disbale after ansible 6.7.0
```
Comment (p-linnane) [inline Formula/ansible@6.rb:12]:
```suggestion
  # disable after ansible 6.7.0
```

File: Formula/ansible@6.rb [hunk 2]
```diff
@@ -0,0 +1,613 @@
...
+  homepage "https://www.ansible.com/"
+  url "https://files.pythonhosted.org/packages/06/75/6e1da76cbc323e5170ac9e63b7c86e17f4387614129ef2514b407c6080d2/ansible-6.6.0.tar.gz"
+  sha256 "e1b940a8d4f412123ede3c14b25cb99c3c8a4d535fd040aabf8e4fb7b0e4f092"
+  license "GPL-3.0-or-later"
+
+  keg_only :versioned_formula
+
+  # disable after ansible 6.7.0
+  # https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-community-package-release-cycle
+  # disable! date: "2023-05-11", because: :unmaintained
```
Comment (SMillerDev) [inline Formula/ansible@6.rb:14]: Shouldn't we deprecate it now then?

Comment (chenrui333) [reply Formula/ansible@6.rb:14]: idk, there would be a new release with 6.7.0, i think we can deprecate/disable at that time.

Comment (SMillerDev) [reply Formula/ansible@6.rb:14]: But it's already deprecated. And after the next release it'll be discontinued.

Comment (chenrui333) [reply Formula/ansible@6.rb]: Sounds good, add a note detailing the `deprecate` reason.

**Comment by Bo98** [comment]
> added a placeholder `disable` line to reminder to disable after `6.7.0` release

Note that our minimum deprecation period is 3 months, and the 6.7.0 release is coming I think next week.

**Comment by chenrui333** [comment]
> > added a placeholder `disable` line to reminder to disable after `6.7.0` release
> 
> Note that our minimum deprecation period before disabling is 3 months (otherwise there's no point shipping a 6.7.0 if it's disabled immediately and people can't upgrade), and the 6.7.0 release is coming I think next week.

yeah, that is what I was thinking to add `disable` ansible@6 at a future time.

**Comment by cho-m** [comment]
Not sure if we want to add a quick fix for #115922 by downgrading `openstacksdk` resource and locking in version within `pypi_formula_mappings.json`. Looks like 7.0.0 is also max bounded:
```console
❯ rg 'MAXIMUM_SDK_VERSION =' ansible-7.0.0/ansible_collections/openstack/cloud/plugins/module_utils/openstack.py
80:MAXIMUM_SDK_VERSION = '0.98.999'

❯ rg 'MAXIMUM_SDK_VERSION =' ansible-6.6.0/ansible_collections/openstack/cloud/plugins/module_utils/openstack.py
80:MAXIMUM_SDK_VERSION = '0.98.999'
```

A followup discussion/fix may be needed on how we want to handle Ansible plugins as we aren't able to track/maintain these well. It is possible that some of the resources may be no longer used, but it is hard to tell as we use `extra_packages`.

---

Based on https://opendev.org/openstack/ansible-collections-openstack#breaking-backward-compatibility-warning, there is a separate release series for newer `openstacksdk` but doesn't seem ready yet.

**Review by SMillerDev — DISMISSED** [review]

**Comment by Tony-Sol** [comment]
Isn't it ready for merge?

**Comment by chenrui333** [comment]
Updating to ansible 7.1.0 and ansible@6 6.7.0 and making the final run.

**Review by iMichka — APPROVED** [review]

**Pull request closed without merging**